### ISSUE TITLE
New IPython Console handling both clients and kernels

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1053]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T08:45:05Z_

Before being able to implement this plugin (which is an evolution of the "IPython Console" plugin introduced with issue #1050 in Spyder v2.2+), we have to implement the features described in blocked issues.

To avoid polluting too much this issue thread, let's add "child" issues to the "Blocked on" list only.

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1053_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T06:45:26Z_

**Labels:** -Priority-Medium -MS-v2.2 Priority-High MS-v2.3  
  **Blockedon:** 1050 1051 1052

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T06:53:33Z_

**Blockedon:** spyderlib:1054

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T06:57:59Z_

**Blockedon:** spyderlib:1055

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T07:08:20Z_

**Blockedon:** spyderlib:1056

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-05-26T08:07:15Z_

**Blockedon:** spyderlib:1057

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-05-26T08:16:01Z_

**Blockedon:** spyderlib:1058

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-26T08:36:48Z_

**Blockedon:** spyderlib:1059

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-27T00:29:26Z_

**Blockedon:** spyderlib:1061

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-27T01:29:03Z_

**Blockedon:** spyderlib:1062

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2012-05-27T04:39:20Z_

Using new issue label "Cat-IPythonConsole".

**Labels:** Cat-IPythonConsole

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-05-30T06:31:37Z_

**Blockedon:** spyderlib:1068

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-05-30T06:33:45Z_

**Blockedon:** spyderlib:1069

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-06-04T18:43:48Z_

**Blockedon:** spyderlib:1078

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-06-08T21:05:06Z_

**Blockedon:** spyderlib:1083

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2012-07-05T20:14:33Z_

**Blockedon:** -spyderlib:1050 -spyderlib:1051 -spyderlib:1052 -spyderlib:1054 -spyderlib:1055 -spyderlib:1056 -spyderlib:1057 -spyderlib:1058 -spyderlib:1059 -spyderlib:1061 -spyderlib:1062 -spyderlib:1068 -spyderlib:1069 -spyderlib:1078 -spyderlib:1083 spyderlib:1050 spyderlib:1051 spyderlib:1052 spyderlib:1054 spyderlib:1055 spyderlib:1056 spyderlib:1057 spyderlib:1058 spyderlib:1059 spyderlib:1061 spyderlib:1062 spyderlib:1068 spyderlib:1069 spyderlib:1078 spyderlib:1083 spyderlib:1085

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2013-02-03T19:52:06Z_

**Labels:** -MS-v2.3 MS-v2.2

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2013-02-11T09:00:04Z_

**Blockedon:** spyderlib:916

**Comment by spyder-issues-migrator** [comment]
_From [jed.lud...@gmail.com](https://code.google.com/u/114333776555678317161/) on 2013-03-05T00:22:11Z_

IPython console and kernel plugin locations will not be changed for v2.2. This will not happen until at least 2.3.

**Labels:** -MS-v2.2 MS-v2.3

**Comment by spyder-issues-migrator** [comment]
_From [jed.lud...@gmail.com](https://code.google.com/u/114333776555678317161/) on 2013-07-03T09:15:17Z_

**Labels:** -MS-v2.3 MS-v2.4

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2015-02-15T16:14:33Z_

**Labels:** -Component-IPython

**Comment by ccordoba12** [comment]
Closing as duplicate of #1860
